app/src/main/java/com/infomaniak/mail/ui/main/menuDrawer/MenuDrawerAdapter.kt

```diff
@@ -110,9 +118,40 @@ class MenuDrawerAdapter @Inject constructor() : ListAdapter<Any, MenuDrawerViewH
             hasCollapsableDefaultFolder = temporaryHasCollapsableDefaultFolder
             hasCollapsableCustomFolder = temporaryHasCollapsableCustomFolder
 
-            // Footer
+            // Actions
             add(ItemType.DIVIDER)
-            add(MenuDrawerFooter(permissions, quotas))
+            add(ItemType.ACTIONS_HEADER)
+            if (areActionsExpanded) {
+                add(
+                    MenuDrawerAction(
```

Comment:
Could we make these three items, static instances values?

Reply:
I must have miss clicked the area where I've put the comment because what I really wanted to target was:
```kt
MenuDrawerAction(
    type = ActionType.IMPORT_MAILS,
    icon = R.drawable.ic_drawer_download,
    text = R.string.buttonImportEmails,
    maxLines = 1,
),
```

Reply:
These three values being more akin to some kind of constant, wouldn't we rather want to name them using all capital letters?